src/book-demo.njk

```diff
@@ -8,12 +8,64 @@ meta:
     description: Book a demo to understand how FlowFuse helps you professionalize your Node-RED deployment.
 ---
 <div class="text-left max-w-full ff-bg-light">
-  <div class="flex justify-center items-center md:block">
-    <div class="ff-bg-light w-full product px-4 pt-4 md:pt-12">
+  <div class="flex flex-col justify-center items-center md:block">
+    <div class="ff-bg-light w-full product px-4 pt-4 md:pt-12 -mb-24 sm:-mb-6">
```

Comment:
Does tailwind have a way to collect classed together into one with a semantical naming scheme?

Reply:
I created new classes, it's cleaner now